frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -15,58 +15,68 @@ import RouteSection from './RouteSection';
-        {values.isi || values.pipeline?.enabled ? null : (
-          <ProgressiveListItem name={t('devconsole~Build configuration')}>
-            <BuildConfigSection
-              namespace={values.project.name}
-              resource={appResources?.buildConfig?.data}
-            />
-          </ProgressiveListItem>
-        )}
-        <ProgressiveListItem name={t('devconsole~Deployment')}>
-          <DeploymentConfigSection
+    <ProgressiveList visibleItems={visibleItems} onVisibleItemChange={handleVisibleItemChange}>
+      <ProgressiveListItem name={t('devconsole~Health checks')}>
+        <HealthChecks title={t('devconsole~Health checks')} resourceType={values.resources} />
+      </ProgressiveListItem>
+      {/* Hide Build for Deploy Image or when a Pipeline is added */}
+      {values.isi || values.pipeline?.enabled ? null : (
+        <ProgressiveListItem name={t('devconsole~Build configuration')}>
+          <BuildConfigSection
             namespace={values.project.name}
-            resource={appResources?.editAppResource?.data}
+            resource={appResources?.buildConfig?.data}
           />
         </ProgressiveListItem>
-        <ProgressiveListItem name={t('devconsole~Scaling')}>
-          {values.resources === Resources.KnativeService ? (
-            <ServerlessScalingSection />
-          ) : (
-            <ScalingSection name="deployment.replicas" />
-          )}
-        </ProgressiveListItem>
-        <ProgressiveListItem name={t('devconsole~Resource limits')}>
-          <ResourceLimitSection />
-        </ProgressiveListItem>
-        <ProgressiveListItem name={t('devconsole~Labels')}>
-          <LabelSection />
-        </ProgressiveListItem>
-      </ProgressiveList>
+      )}
+      <ProgressiveListItem name={t('devconsole~Deployment')}>
+        <DeploymentConfigSection
+          namespace={values.project.name}
+          resource={appResources?.editAppResource?.data}
+        />
+      </ProgressiveListItem>
+      <ProgressiveListItem name={t('devconsole~Scaling')}>
+        {values.resources === Resources.KnativeService ? (
+          <ServerlessScalingSection />
+        ) : (
+          <ScalingSection name="deployment.replicas" />
+        )}
+      </ProgressiveListItem>
+      <ProgressiveListItem name={t('devconsole~Resource limits')}>
+        <ResourceLimitSection />
+      </ProgressiveListItem>
+      <ProgressiveListItem name={t('devconsole~Labels')}>
+        <LabelSection />
+      </ProgressiveListItem>
+    </ProgressiveList>
+  );
+};
+
+const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values, appResources }) => {
+  const { t } = useTranslation();
+
+  return (
+    <FormSection title={t('devconsole~Advanced options')}>
+      <RouteSection route={values.route} resources={values.resources} />
+      <div>
+        <Trans ns="devconsole">
+          Click on the names to access advanced options for{' '}
```

Comment:
When using this string with the space included, the i18n tools export can export the same string for all three cases.

```suggestion
          {'Click on the names to access advanced options for '}
```